demo/style.css

```diff
@@ -0,0 +1,9 @@
+body {
+  background-color: #6e6d6d;
+}
+
+#player-placeholder {
+  margin: 100px auto;
+  max-width: 640px;
+  /*aspect-ratio: 10 / 3;*/
```

Comment:
commented leftovers

Reply:
Intently there for me - it is refs for me which are critically relevant  for the next task on mini-audio-responsivness